lightning-background-processor/src/lib.rs

```diff
@@ -937,7 +940,20 @@ where
 					.await
 			};
 			// TODO: Once our MSRV is 1.68 we should be able to drop the Box
-			futures.set_a(Box::pin(fut));
+			let mut fut = Box::pin(fut);
+
+			// Because persisting the ChannelManager is important to avoid accidental
+			// force-closures, go ahead and poll the future once before we do slightly more
+			// CPU-intensive tasks in the form of NetworkGraph pruning or scorer time-stepping
+			// below. This will get it moving but won't block us for too long if the underlying
```

Comment:
If this is needed, shouldn't we stay on the safe side and await cm persistence separately? (no include it in the joiner)